tests/bin/check_tron_jobs_test.py

```diff
@@ -439,6 +454,11 @@ def test_most_recent_job_with_smaller_run_id_failed(self):
                             '%Y-%m-%d %H:%M:%S',
                             time.localtime(time.time() - 1800),
                         ),
+                    'end_time':
+                        time.strftime(
+                            '%Y-%m-%d %H:%M:%S',
+                            time.localtime(time.time() - 1700),
```

Comment:
Maybe this test should be called most_recent_end_time, since the run ID doesn't matter?

Reply:
Sure, that makes sense.